src/Storages/ObjectStorageQueue/ObjectStorageQueueIFileMetadata.cpp

```diff
@@ -322,6 +322,13 @@ void ObjectStorageQueueIFileMetadata::setFailed(const std::string & exception_me
             throw;
         }
     }
+    else
+    {
+        resetProcessing();
```

Comment:
I don't see `resetProcessingImpl` implemented for `ObjectStorageQueueOrderedFileMetadata` :eyes:

Reply:
Good point